pkg/apis/flowcontrol/validation/validation.go

```diff
@@ -564,3 +609,10 @@ func memberInList(seek string, a ...string) bool {
 	}
 	return false
 }
+
+func isNominalConcurrencyShareFieldDefault(obj *flowcontrol.PriorityLevelConfiguration) bool {
```

Comment:
this is a confusing name... I'd suggest something like:

```go
func hasZeroNominalConcurrencyShares(obj *flowcontrol.PriorityLevelConfiguration) bool {
  return obj != nil && obj.Spec.Limited != nil && obj.Spec.Limited.NominalConcurrencyShares == 0
}
```

(the logic gets simpler when the internal  NominalConcurrencyShares field goes back to being a non-pointer)